src/webportal/src/app/job/job-view/fabric/job-detail.jsx

```diff
@@ -215,8 +226,73 @@ class JobDetail extends React.Component {
               onStopJob={this.stop}
               onReload={this.reload}
             />
-            {this.renderTaskRoles()}
-          </div>
+            <Card>
+              <Stack gap='l2' padding='l2'>
+                <Stack horizontal gap='m' verticalAlign='center'>
+                  <Text variant='large'>Job Attempt Index</Text>
+                  <Dropdown
+                    styles={{ root: { width: '150px' } }}
+                    placeholder='Select Attempt Index'
+                    options={attemptIndexOptions}
+                    defaultSelectedKey={selectedAttemptIndex || undefined}
+                    onChange={this.onChangeJobAttempt}
+                  />
+                </Stack>
+                <Stack
+                  horizontal
+                  horizontalAlign='space-between'
+                  verticalAlign='end'
+                  gap='m'
+                >
+                  <Stack horizontal gap='l1'>
+                    <Stack gap='m'>
+                      <Text>Attempt State</Text>
+                      <StatusBadge
+                        status={capitalize(jobInfo.jobStatus.attemptState)}
+                      />
+                    </Stack>
+                    <Stack gap='m'>
+                      <Text>Attempt Start Time</Text>
+                      <Text>
+                        {DateTime.fromMillis(
+                          jobInfo.jobStatus.createdTime,
```

Comment:
>createdTime [](start = 44, length = 11)

Attempt Start Time should be jobInfo.jobStatus.appLaunchedTime

Reply:
done